README.md

```diff
@@ -278,6 +278,24 @@ API. The commented interface is available in
 and
 [http](src/python/library/tritonclient/http/__init__.py).
 
+### GRPC KeepAlive Options
+
+Triton exposes GRPC KeepAlive parameters with the default values for both
+client and server described [here](https://github.com/grpc/grpc/blob/master/doc/keepalive.md).
+
+You can find a `KeepAliveOptions` struct/class that encapsulates these
+parameters in both the [C++](src/c%2B%2B/library/grpc_client.h) and
+[Python](src/python/library/tritonclient/grpc/__init__.py) client libraries.
+
+There is also a [C++](src/c%2B%2B/examples/simple_grpc_keepalive_client.cc) and
+[Python](src/python/examples/simple_grpc_keepalive_client.py) example
+demonstrating how to setup these parameters on the client side. For a server
+side example, refer to [main.cc](https://github.com/triton-inference-server/server/blob/main/src/servers/main.cc).
```

Comment:
Instead of pointing to main.cc I think it will be better to point to the added section with [this](https://github.com/triton-inference-server/server/pull/3149) PR.